lib/mpl_toolkits/axes_grid1/axes_divider.py

```diff
@@ -291,14 +291,14 @@ def add_auto_adjustable_area(self, use_axes, pad=0.1, adjust_dirs=None):
 class AxesLocator:
     """
     A callable object which returns the position and size of a given
-    AxesDivider cell.
+    `AxesDivider` cell.
```

Comment:
```suggestion
    `.AxesDivider` cell.
```
I recommend to always keep the leading dot. Without the dot, the class is only searched for in the current module. While that would be sufficient here, it can still have surprising side effects. For example, if you derive from `AxesLocator` in another module and inherit the docstring, the reference would not resolve for the child class.